src/third-party/PhishingController.test.ts

```diff
@@ -26,35 +26,69 @@ describe('PhishingController', () => {
 
   it('should set default config', () => {
     const controller = new PhishingController();
-    expect(controller.config).toStrictEqual({ interval: 3_600_000 });
-  });
-
-  it('should poll and update rate in the right interval', async () => {
-    await new Promise<void>((resolve) => {
-      const mock = sinon.stub(
-        PhishingController.prototype,
-        'updatePhishingLists',
-      );
-      new PhishingController({ interval: 10 });
-      expect(mock.called).toBe(true);
-      expect(mock.calledTwice).toBe(false);
-      setTimeout(() => {
-        expect(mock.calledTwice).toBe(true);
-        resolve();
-      }, 15);
-    });
+    expect(controller.config).toStrictEqual({ refreshInterval: 3600000 });
   });
 
-  it('should clear previous interval', async () => {
-    const mock = sinon.stub(global, 'clearTimeout');
-    const controller = new PhishingController({ interval: 1337 });
-    await new Promise<void>((resolve) => {
-      setTimeout(() => {
-        controller.poll(1338);
-        expect(mock.called).toBe(true);
-        resolve();
-      }, 100);
-    });
+  it('does not fetch or do anything async upon construction', async () => {
```

Comment:
Looks like we use "should do XYZ" in this test instead of "does XYZ / does not do XYZ", what do you think about using this style for the test names?